### PR TITLE
correct text, delete map, correct contacts, add links

### PR DESCRIPTION
**Pull request by AlexeyDolya**

### CODE REVIEW

File: client/app/contacts/contacts.component.html [hunk 1]
```diff
@@ -75,17 +67,16 @@ <h6>fill out an application</h6>
         <div class="tel-block">
           <ul>
             <li class="tel">
-              <a href="tel:+35623157586">+38 (096) 226 40 32</a>
+              <p>Skype:</p>
```
Comment (LimarenkoDenis) [inline client/app/contacts/contacts.component.html]: add mail to  (skype)

File: markup/sass/_contacts.sass [hunk 2]
```diff
@@ -168,12 +168,6 @@
       .tel
         text-align: center
         margin-bottom: 10px
-        a
```
Comment (LimarenkoDenis) [inline markup/sass/_contacts.sass]: return back

File: client/app/header/header.component.html [hunk 3]
```diff
@@ -9,12 +9,13 @@
       <div class="navigation-wrapper" [ngClass]="{'open-menu': isOpen}">
         <nav>
           <ul id="menu">
-            <li *ngFor="let menu of menu">
-              <a  [href]="menu.link"
-                  (click)="navigate(menu.title)"
-                  [ngClass]="{'active': (activeMenu$ | async) === menu.title}">
-                {{menu.title}}
-              </a>
+            <li *ngFor="let m of menu">
```
Comment (NepipenkoIgor) [inline client/app/header/header.component.html]: m -> menuItem

File: client/app/header/header.component.html [hunk 4]
```diff
@@ -9,13 +9,13 @@
...
-            <li *ngFor="let m of menu">
-                <ng-template [ngIf]="!m.href" [ngIfElse]="simple_link">
-                    <a appScrollAnchor [component]="m.component" >{{m.name}}</a>
-                  </ng-template>
-                  <ng-template #simple_link>
-                    <a [href]="m.href" [target]="m.target">{{m.name}}</a>
-                  </ng-template>
+            <li *ngFor="let menuItem of menu">
+              <ng-template [ngIf]="!menuItem.href" [ngIfElse]="simple_link">
+                <a appScrollAnchor [component]="menuItem.component" (click)="navigate(menu.title)">{{menuItem.name}}</a>
```
Comment (LimarenkoDenis) [inline client/app/header/header.component.html]: return back ngClass

**Pull request merged**